config/stored_requests.go

```diff
@@ -185,7 +185,9 @@ func (cfg *PostgresConfig) validate(dataType DataType, errs configErrors) config
 		return errs
 	}
 
-	return cfg.PollUpdates.validate(dataType, errs)
+	errs = cfg.CacheInitialization.validate(dataType, errs)
```

Comment:
That is always annoying to miss a validation routine. I guess since the type had a different name from the struct property, it was harder to catch.